src/page-templates/content-type/content-type.tsx

```diff
@@ -75,7 +102,28 @@ const ContentTypePage: NextPage<ContentTypePageProps> = ({
         searchString,
         setSearchString,
         numberOfResults,
-    } = useSearch(contentType);
+    } = useSearch(
+        contentType,
+        undefined,
+        undefined,
+        initialSearchData ? pageNumber : undefined
+    );
+
+    const hasEmptyFilterAndQuery = () => {
+        return (!searchString || searchString == '') && allFilters.length == 0;
+    };
+
+    const getResultData = () => {
+        return initialSearchData && hasEmptyFilterAndQuery()
+            ? initialSearchData
+            : data;
+    };
+
+    const getResultIsValidating = () => {
```

Comment:
if there's project-wide util files available, these can be combined with the functions in `search.tsx`

Reply:
@justinmckenzie These functions are repeated but I don't think there will be much gain to moving them to a utils because the dependencies are specific to each component (e.g. isValidating, initialSearchContent, data, searchString, allFilters) and each component is technically repeated but slightly different, so I would need to pass 5 parameters and we'd only be repeating ternary operator logic. I did however, move `hasEmptyFilterAndQuery` to the hook utilities file.

Reply:
Ended up moving these out to utils anyway, but added a note for this to be refactored at a later time due to the amount of parameters. I think this might actually be removed in the refactor PR mentioned above.

Reply:
@ajhenri sounds good...i was just making the suggestion because I noticed the repetition, if it doesn't really make sense to do it I'm totally cool with whatever approach you think is best.

Reply:
Haha sure yeah, it was already done. Does look much better without the repetition so it was a great suggestion